src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.AreaCalculator;
+import core.basesyntax.InfoPrinter;
+
+public abstract class Figure implements AreaCalculator, InfoPrinter {
+    private String name;
+    private String color;
```

Comment:
Better use **Color** type instead **String** for your color property. Using String here you have a risk to create a figure with the color "My cat" for example.